script.js

```diff
@@ -0,0 +1,95 @@
+const container = document.querySelector(".container");
+const resetBtn = document.querySelector(".reset");
+const gridSize = document.querySelector(".grid-size");
+const rainbowBtn = document.querySelector(".rainbow");
+const blackBtn = document.querySelector(".black");
+const eraseBtn = document.querySelector(".erase");
+let isBlack = true;
+
+rainbowBtn.addEventListener("click", () => {
+  let color = "rainbow";
+  setColor(color);
```

Comment:
Any need for creating a (local) variable for the colors? You could have `setColor("rainbow")`

Same for the other two below.